i18n/fr.jsonp.js

```diff
@@ -178,19 +179,19 @@ document.localeJson = {
-      "dialog-serviceworker-unsupported-fallback": "Utiliser le mode JQuery",
+      "dialog-serviceworker-unsupported-message": "<p>Malheureusement, votre navigateur ne semble pas prendre en charge le mode ServiceWorker, qui est maintenant le mode par défaut pour cette application.</p><p>Vous pouvez continuer à utiliser l'application en mode Sécurisé, mais veuillez noter que ce mode ne fonctionne bien qu'avec les fichiers ZIM qui ont un contenu statique, comme les fichiers ZIM de Wikipédia / WikiMédia ou Stackexchange.</p><p>Si possible, nous vous recommandons de mettre à jour votre navigateur vers une version prenant en charge le mode ServiceWorker.</p>",
+      "dialog-serviceworker-unsupported-fallback": "Utiliser le mode Sécurisé",
       "dialog-serviceworker-defaultmodechange-title": "Changement du mode d'injection de contenu par défaut",
-      "dialog-serviceworker-defaultmodechange-message": "<p>Nous vous avons basculé en mode ServiceWorker (c'est maintenant le mode par défaut). Il prend en charge plus de types de fichiers ZIM et est beaucoup plus robuste.</p><p>Si vous rencontrez des problèmes avec ce mode, vous pouvez revenir au mode JQuery (maintenant obsolète). Dans ce cas, veuillez signaler les problèmes que vous avez rencontrés (voir la page Informations).</p>",
+      "dialog-serviceworker-defaultmodechange-message": "<p>Nous vous avons basculé en mode ServiceWorker (c'est maintenant le mode par défaut). Il prend en charge plus de types de fichiers ZIM et est beaucoup plus robuste.</p><p>Si vous rencontrez des problèmes avec ce mode, vous pouvez revenir au mode Sécurisé. Dans ce cas, veuillez signaler les problèmes que vous avez rencontrés (voir la page Informations).</p>",
       "dialog-slideawaycheck-message": "Ce paramètre peut ne pas fonctionner correctement chez Firefox OS. Si vous constatez que certains liens ZIM deviennent inactifs, veuillez désactiver ce paramètre.",
-      "dialog-sourceverification-alert": "Cette archive ZIM provient-elle d'une source fiable&nbsp;? Si ce n'est pas le cas, vous pouvez toujours lire le fichier ZIM en mode sécurisé (également appelé mode JQuery). La fermeture de cette fenêtre ouvre également le fichier en mode sécurisé. Cette option peut être désactivée dans Options avancées.",
+      "dialog-sourceverification-alert": "Cette archive ZIM provient-elle d'une source fiable&nbsp;? Si ce n'est pas le cas, vous pouvez toujours lire le fichier ZIM en mode Sécurisé (également appelé mode Sécurisé). La fermeture de cette fenêtre ouvre également le fichier en mode Sécurisé. Cette option peut être désactivée dans Options avancées.",
```

Comment:
Delete " (également appelé mode Sécurisé)". This is the old "aka JQuery Mode", which you've changed instead of deleting.